City-of-Helsinki/drupal-hdbt
# Uhf 3233 fix finnish quote

* Fixed quotation to follow [Finnish quotation formatting](https://jkorpela.fi/suomi/lain.html). 
* Fixed scss nesting problems

## How to test

* Get branch
* Check a page in Finnish with a quotation, it should start with `”`
* Check a page in English with a quotation, it should start with `“`
* Check also code improvements from https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/100
